i18n/en.json

```diff
@@ -1498,7 +1498,7 @@
   },
   {
     "id": "api.plugin.add_public_key.not_a_public_key.app_error",
-    "translation": "Public key should have the extension .asc"
+    "translation": "Public key should have the extension .plugin.asc"
```

Comment:
Is this true for the code that was previously using this message?

Reply:
I'm not sure I understood the question. We are forcing the user to have plugin signatures stored in `*.plugin.asc` files

Reply:
I was referring to where this message is being used that the new message still apply. However, these messages does not seem to be referenced anywhere? 🤔